### PR TITLE
Add mail support to pythonexpress

### PR DESCRIPTION
**Pull request by kracekumar**

### CODE REVIEW

File: salt/roots/mail/init.sls [hunk 1]
```diff
@@ -19,23 +19,26 @@ mail-pkgs:
...
+    - source: salt://mail/exim4/virtual/mail_aliases
     - template: jinja
     - defaults:
         mail_alias: {{ pillar['mail-alias'] }}
 
+{% if pillar['mail-alias'] == 'in.pycon.org' %}
 /etc/exim4/virtual/pycon.pssi.org.in:
   file.symlink:
-    - target: /etc/exim4/virtual/in.pycon.org
+    - target: /etc/exim4/virtual/mail_aliases
```
Comment (vigneshsarma) [inline salt/roots/mail/init.sls]: that line is for the server. Not sure what you are doing.

Comment (kracekumar) [inline salt/roots/mail/init.sls]: Thanks for the catch. Fixed it.

File: salt/roots/mail/init.sls [hunk 2]
```diff
@@ -19,23 +19,26 @@ mail-pkgs:
...
+    - source: salt://mail/exim4/virtual/mail_aliases
     - template: jinja
     - defaults:
         mail_alias: {{ pillar['mail-alias'] }}
 
+{% if pillar['mail-alias'] == 'in.pycon.org' %}
 /etc/exim4/virtual/pycon.pssi.org.in:
   file.symlink:
-    - target: /etc/exim4/virtual/in.pycon.org
+    - target: /etc/exim4/virtual/{{ pillar['mail-alias'] }}
```
Comment (vigneshsarma) [inline salt/roots/mail/init.sls]: really?

have you looked at what the key stores?

Comment (kracekumar) [inline salt/roots/mail/init.sls]: Dang! you're right. Thanks and fixed.

**Pull request merged**